Library/Homebrew/cask/cask.rb

```diff
@@ -122,6 +122,21 @@ def outdated_versions(greedy = false)
       installed.reject { |v| v == version }
     end
 
+    def outdated_info(greedy, verbose, json)
+      if json
+        {
+          name:               token,
+          installed_versions: outdated_versions(greedy).join(", "),
```

Comment:
Thoughts about splitting `outdated_versions(greedy).join(", ")` into a variable?

Reply:
Done! I also moved the --quiet case to its own early return clause.